lib/generators/blacklight/solr4_generator.rb

```diff
@@ -10,10 +13,8 @@ class Solr4Generator < Rails::Generators::Base
     EOS
 
     def install_jettywrapper
-      return unless options[:jettywrapper]
```

Comment:
Yeah, I could see that though.

If you do get there, and you did not pass the jettywrapper options, you'll simply execute the following code: https://github.com/projectblacklight/blacklight/blob/master/lib/generators/blacklight/solr4_generator.rb#L23-L25, which doesn't seem particularly useful to me.

I also experimented w/ passing this options directly on w/o much luck.